app/page.tsx

```diff
@@ -102,31 +102,31 @@ export default function Home() {
             <div className="flex text-center text-[40px] mb-10">Connect database...</div>
             <div className="flex flex-row items-center justify-between w-full mb-10">
```

Comment:
`flex flex-row items-center justify-between w-full mb-10`
Can you possibly remove the duplicated classnames?